client/root-error-boundary.tsx

```diff
@@ -82,14 +84,16 @@ export class RootErrorBoundary extends React.Component<
           <WindowControlsStyle />
           <WindowControls />
           <Container>
-            <Headline5>Something went wrong :(</Headline5>
+            <Headline5>{t('rootErrorBoundary.title', 'Something went wrong :(')}</Headline5>
```

Comment:
Given that we know an exception has already occurred and we don't know what state things are in, I think you probably want to try and be *very safe* with making calls into i18next here. Calls should probably be wrapped in a try/catch and fallback to the English values on failure